guides/source/active_record_querying.md

```diff
@@ -85,7 +85,7 @@ The methods are:
 
 All of the above methods return an instance of `ActiveRecord::Relation`.
 
-The primary operation of `Model.find(options)` can be summarized as:
+The primary operation of `Model.finder_method(options)` can be summarized as:
```

Comment:
I think this is confusing, as it implies that `finder_method` is a method name in Rails.